build/buildBuildImages.sh

```diff
@@ -432,6 +472,8 @@ elif [ "$imageTypeToBuild" == "ltsversions" ]; then
 	buildLtsVersionsImage
 elif [ "$imageTypeToBuild" == "ltsversions-buster" ]; then
 	buildLtsVersionsImage "buster"
+elif [ "$imageTypeToBuild" == "all" ]; then
```

Comment:
Same comment above -- when updating everything to `latest`, include the changes down here